wpeverest/everest-forms
# Fix - Delete uploaded files after removing Forms or Entries

### All Submissions:






### Changes proposed in this Pull Request:

Previously the files were uploaded to the server after creating the form. After deleting entries or form, the files was unable to remove. This PR helps to remove unnecessary files after form or entries has been removed from the system.


### How to test the changes in this Pull Request:

1. Open the **Form Builder** area and choose Image Upload or Form Upload fields.
2. Save the changes and go to the **Preview** area.
3. Fill up the form and upload images or files in the form.
4. Save changes and remove any or entry related to image or file upload. You can remove Forms as well which contains the image or file attachments.
5. You can also upload the files using **PDF Form Submission** Add-on.
6. Look in the folder where the files have been uploaded. Any files related to forms have been successfully removed.
7. That's all. You will see now clear folders not containing unwanted files or folders.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Delete uploaded files after removing Forms or Entries.
